kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-sample/src/test/java/org/kie/server/springboot/samples/KafkaConsumerHappyPathTest.java

```diff
@@ -135,17 +166,17 @@ public static void beforeClass() {
     
     @Before
     public void setup() {
-        unit = setup(deploymentService, SIGNALLING_PROJECT);
+        unit = kafkaFixture.setup(deploymentService, SIGNALLING_PROJECT, SINGLETON.name());
```

Comment:
Not sure here - would it make sense to also parameterize the strategy for the KafkaConsumer?

Reply:
Not the goal of this PR